packages/lesswrong/components/ea-forum/giving-portal/EAGivingPortalPage.tsx

```diff
@@ -211,7 +211,6 @@ const styles = (theme: ThemeType) => ({
     flexWrap: "wrap",
     gap: "16px",
     rowGap: "16px",
-    justifyContent: "center",
```

Comment:
I made this change to the "Other opportunities" section, so that when there is a lone card in a row it's not awkwardly in the middle, and instead is on the left side of the grid.

<img width="1067" alt="Screen Shot 2023-11-02 at 3 24 46 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/6314cbb8-b57e-4106-8ca8-35af45fdcb3d">